Makefile.build

```diff
@@ -195,7 +195,7 @@ TIMER=$(if $(TIMED), $(STDTIME), $(TIMECMD))
 #   TIME="%C (%U user, %S sys, %e total, %M maxres)"
 
 COQOPTS=$(NATIVECOMPUTE) $(COQWARNERROR) $(COQUSERFLAGS)
-BOOTCOQC=$(TIMER) $(COQTOPBEST) -boot $(COQOPTS) -compile
+BOOTCOQC=$(TIMER) $(COQTOPBEST) -boot $(COQOPTS) -_compile
```

Comment:
Why not use coqc here btw?

Reply:
That's delayed to #8690 as to avoid modifying the make rules too much and thus introducing a regression here. In current `master`, `COQC` doesn't depend on `coqtop` (!!) so if we modify the rule `coqc` will be called right away and fail as `coqtop` has not been build.

As I wanted to introduce the least amount of functional changes in this PR I didn't perform this deep make modification here; if we end up skipping this PR then this would be solved.

Reply:
That's indeed another bug of the make system I think.

Reply:
Same happens for the Dune rules, etc...

Reply:
Done.